### ISSUE TITLE
Text preceded by an /r/n (carriage return + linefeed) causes REGEX parser to fail

### ISSUE THREAD

**Issue by James-A-White** [Cretezy/linkify#15]
Hello!

Great plugin! Saved me so much time fiddling with text spans!!!!

I ran into one problem, I'm pulling text from a database connection and the text is returned with carriage returns and linefeeds, which seems to make the REGEX parser fail. To fix it, I do a text replace of all '\r\n' to simply '\n' and Linkify works perfectly.

Might be something to look at in a future release, but in the interim the fix is easy as in the below code:

```
Linkify(
  text: widget.kennel['kennelDescription'].toString().replaceAll('\r\n', '\n'),
  style: bodyStyle,
  linkStyle: bodyStyleYellow,
  humanize: true,
  onOpen: (LinkableElement link) async {
    if (await canLaunch(link.url)) {
      await launch(link.url);
    } else {
      Utilities.showAlert(context, 'Unable to open link', 'The app was unable to open ${link.url}', 'OK');
    }
  },
),

```

**Comment by Vov4yk** [comment]
Any luck with solution?
My work around 
    .replaceAll("\r\n", '\n'),

**Comment by Cretezy** [comment]
This has been resolved with #26 in https://github.com/Cretezy/linkify/commit/63e97d597518509baf60b90e44ef828096902e69
